Gemfile

```diff
@@ -23,11 +23,11 @@ gem "rails", "~> 7.0.0"
 gem "raygun4ruby"
 gem "redis"
 gem "sassc-rails"
+gem "shakapacker", "~> 7.0"
```

Comment:
note: I've taken us just to v7 for now, because v8 doesn't have a lot of breaking changes and will be a lot easier to do once we've on v7